tokio/src/runtime/builder.rs

```diff
@@ -554,6 +590,54 @@ impl Builder {
         self
     }
 
+    /// Sets the number of scheduler ticks after which the scheduler will poll the global
+    /// task queue.
+    ///
+    /// A scheduler "tick" roughly corresponds to one `poll` invocation on a task. Schedulers
+    /// have a local queue of already-claimed tasks, and a global queue of incoming tasks.
+    /// Setting the interval to a smaller value increases the fairness of the scheduler,
+    /// at the cost of more synchronization overhead.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use tokio::runtime;
+    ///
+    /// # pub fn main() {
+    /// let rt = runtime::Builder::new_multi_thread()
+    ///     .global_queue_interval(31)
+    ///     .build();
+    /// # }
+    /// ```
+    pub fn global_queue_interval(&mut self, val: u8) -> &mut Self {
+        self.global_queue_interval = val;
+        self
+    }
```

Comment:
I don't think `u8` is the right type for this. Something like `u32` would be more appropriate. We can change the internals to use an `u32` for the tick.